src/Duke.java

```diff
@@ -0,0 +1,225 @@
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class Duke {
+
+    private static int Count = 0;
+    private static Task[] List = new Task[10]; //creating list of Task for inputs
+    private static Boolean isBye = false;
+
+
+    public static void main(String[] args) {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke");
+
+        try {
+            printFileContents("src/tasks.txt");
+        } catch (FileNotFoundException e) {
+            System.out.println("tasks.txt file not found");
+        }
+
+        System.out.println("What can I do for you?");
+
+        Scanner in = new Scanner(System.in);
+
+        while (!isBye) {
+            String line = in.nextLine();
+
+            if (line.equals("bye")) {
+                isBye = true;
+                break;
+            }//end programme
```

Comment:
Comments should be indented relative to their position in the code. Could refer to the coding standard